.github/workflows/metadata-parser-build-validation.yml

```diff
@@ -0,0 +1,25 @@
+name: Metadata Parser Build Validation
+
+on:
+  push:
+    branches: [ master, dev ]
+  pull_request:
+    branches: [ master, dev ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+      with:
+        submodules: recursive
+    - name: Setup .NET
+      uses: actions/setup-dotnet@v1
+      with:
+        dotnet-version: 5.0.x
+    - name: Restore dependencies
+      run: dotnet restore ./tools/MetadataParser/MetadataParser.csproj
+    - name: Build
+      run: dotnet build ./tools/MetadataParser/MetadataParser.csproj --no-restore
```

Comment:
We could run it for beta and for v1 as well to get feedback on the metadata changes

Reply:
well, those files are auto generated and shouldn't be manually updated. And I have already added a step in generation pipeline to block checking in if it fails.

Reply:
I was thinking about the case where somebody introduces a new transformation in a PR

Reply:
I checked the last 10 XSLT changes, nobody has updated the clean metadata together with those changes, updates were picked up in subsequent full generations.

I also thought about putting this as a gate here and kicking off a PR from the full generation pipeline instead of directly checking into master, but that changes dependencies and structure of the pipeline a lot.

Reply:
Maybe we can consider running it for the test metadata which is often updated with XSLT.

Reply:
I have just run it against the test metadata, and it failed to parse :D

Reply:
git blame! :D
I don't want to be annoying, it's just that I don't trust myself. The faster a machine can check my work and tell me I forgot something/did something wrong, the better :)